packages/@react-spectrum/badge/docs/Badge.mdx

```diff
@@ -0,0 +1,106 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import docs from 'docs:@react-spectrum/badge';
+import {HeaderInfo, PropTable, PageDescription} from '@react-spectrum/docs';
+import packageData from '@react-spectrum/badge/package.json';
+
+```jsx import
+import {Badge} from '@react-spectrum/badge';
+```
+
+---
+category: Status
+keywords: [badge]
+---
```

Comment:
add `after_version: 3.0.0` so that this doesn't go out to the prod docs website right away. it would break until we publish the package.